server/libs/domain/src/facial-recognition/facial-recognition.services.ts

```diff
@@ -94,10 +95,10 @@ export class FacialRecognitionService {
     const output = join(outputFolder, `${personId}.jpeg`);
     this.storageRepository.mkdirSync(outputFolder);
 
-    const left = x1;
-    const top = y1;
-    const width = x2 - x1;
-    const height = y2 - y1;
+    const left = x1 - 30 > 0 ? x1 - 30 : x1;
+    const top = y1 - 30 > 0 ? y1 - 30 : y1;
+    const width = x2 - x1 + 60;
+    const height = y2 - y1 + 60;
```

Comment:
Good enough for now, but I think we should make this a little bit more robust and "smart" in the future. It'd be great if the aspect ratio of the thumbnail was always 1:1, zoomed out a bit, and never overflow the bounds of the image.